src/base/bittorrent/torrentimpl.cpp

```diff
@@ -2959,3 +2963,12 @@ QFuture<std::invoke_result_t<Func>> TorrentImpl::invokeAsync(Func &&func) const
 
     return future;
 }
+
+void TorrentImpl::setComment(const QString &comment)
+{
+    if (m_comment != comment)
+    {
+        m_comment = comment;
+        deferredRequestResumeData();
+    }
+}
```

Comment:
This function should be moved to below `TorrentImpl::comment()`. Like the declaration in torrent.h